docs/canary-config.md

```diff
@@ -94,6 +95,32 @@ See the [Netflix Automated Canary Analysis Judge] and [Mann Whitney Classifier]
 - `criticalIncrease` **5.0** (number, optional) - Defaults to 1. The multiplier increase that must be met for the metric to be a critical failure and fail the entire analysis with a score of 0. This example make the canary fail critically if there is a 5x increase.
 - `criticalDecrease` **0.5** (number, optional) - Defaults to 1. The multiplier decrease that must be met for the metric to be a critical failure and fail the entire analysis with a score of 0. This example make the canary fail critically if there is a 50% decrease.
 
+## Outliers
+
+Controls how to classify and handle outliers.
+Outliers are determined based on the interquartile range (IQR) which is the middle 50% of the data.
```

Comment:
It might be useful to expand on this statement with something along the lines of:
> Conceptually, the IQR measures where the bulk of the values lie within a distribution